### DISCUSSION TITLE
Compact Dashboard by removing "Statistics" and 4 row "Recent Activity"?

### DISCUSSION THREAD

**Discussion by raunz** [ngoduykhanh/PowerDNS-Admin#1417] [Ideas]
Started off from an idea by @AzorianMatt (https://github.com/PowerDNS-Admin/PowerDNS-Admin/pull/1379#issuecomment-1438963873 ) to strip off invaluable items from Dashboard. Todays latest UI is so "large", that the "Statistics" and "Recent Activity" boxes consume 50% of entire dashboard, without revealing any necessary data.
The Statistics would be nice to move a separate page in the Administration menu section. If wanted, operators and admins can have look there. "Recent Activity" is already covered in /admin/history page.
```
821 Domains
450473 Histories
109 Users
2W 5D 3h 48m Uptime
```
I'm in great favor to see the maximum zones list in major area of screen (currently I can see only 6 rows with 1200px vertical resolution without scrolling).

Are there others who would like to compact the dashboard?

Btw. There's a minor bug in /routes/dashboard.py for standard users when the "allow_user_view_history" is OFF - the number of domains is 0, but actually there's a list of zones below... (incomplete if clause)

**Comment by subbink** [comment]
It seems to me that this has been done in the [v0.4.0 release](https://github.com/PowerDNS-Admin/PowerDNS-Admin/releases/tag/v0.4.0).

**Comment by AzorianMatt** [comment]
@raunz as previously discussed, these two blocks have been removed from the dashboard which will now serve as nothing more than a zone list page.

**Reply by raunz** [reply]
Thanks! This topic is solved :)
Nice to see the new dashboard screenshot on the project index page too.
